srcpkgs/praat/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'praat'
 pkgname=praat
-version=6.2.04
+version=6.2.07
```

Comment:
.08 seems to be out

Reply:
Hmmm, they mention 6.2.08 on their homepage, but all downloads still point to 6.2.07.
I guess the tarballs for .08 will be out soon, so let's just wait... :)

Reply:
6.2.09 is out and this time there is a release tarball ;)